test_common/harness/testHarness.cpp

```diff
@@ -819,6 +819,8 @@ test_status callSingleTestFunction(test_definition test,
         if (!context)
         {
             print_error(error, "Unable to create testing context");
+            gFailCount++;
```

Comment:
Elsewhere in this harness we only increment `gTestsFailed` when we return `TEST_FAIL`.  Is there some reason we increment both `gFailCount` and `gTestFailed` for these failures?